src/Stratis.Bitcoin.Features.Consensus/PosBlockValidator.cs

```diff
@@ -12,7 +12,7 @@ public static bool IsCanonicalBlockSignature(PosBlock block, bool checkLowS)
 
             return checkLowS ?
```

Comment:
Seems this check is not similar to whats in X

https://github.com/stratisproject/stratisX/blob/master/src/main.cpp#L2161

Reply:
Also seems to me we now only call IsCanonicalBlockSignature once with boolan to check lowS

Reply:
As discussed, `IsDERSignature` is named `IsValidSignatureEncoding` in SBFN. Or was it a different part of the check you're referring to?